.github/workflows/build-deploy.yml

```
@@ -0,0 +1,53 @@
+
+    steps:
+    # Step 1: Checkout the repository
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    # Step 4: Set up GCP Credentials (Using GitHub Secrets)
+    - name: Set up GCP credentials
+      uses: google-github-actions/setup-gcloud@v2
+      with:
+        project_id: delta-prop-shop
+        credentials: ${{ secrets.GCP_CREDENTIALS }}
```

actionlint (1.7.4)

input "credentials" is not defined in action "google-github-actions/setup-gcloud@v2". available inputs are "install_components", "project_id", "skip_install", "version"

(action)